client/src/App.vue

```diff
@@ -369,6 +370,25 @@ export default {
       this.numReady += 1;
       this.getRangeData(event);
     },
+
+    fetchMovie(id) {
+      axios({
+        url: `http://localhost:5000/api/movie/${id}`,
```

Comment:
We should have some way to configure this at build time. No sure if vue has support for this?